openstates/tn/bills.py

```diff
@@ -229,13 +227,20 @@ def scrape_bill(self, term, bill_url, bill_type):
         # checking if there is a matching bill
         if secondary_bill_id:
             secondary_bill_id = secondary_bill_id[0].text
-
             # swap ids if * is in secondary_bill_id
             if '*' in secondary_bill_id:
                 bill_id, secondary_bill_id = secondary_bill_id, bill_id
                 secondary_bill_id = secondary_bill_id.strip()
-
-        bill_id = bill_id.replace('*', '').strip()
+            secondary_bill_id = secondary_bill_id.replace('  ',' ')
+            
+        bill_id = bill_id.replace('*', '').replace('  ',' ').strip()
+
+        if 'B' in bill_id:
+            bill_type = 'bill'
+        elif 'JR' in start_abbr:
+            bill_type = 'joint resolution'
+        else:
```

Comment:
That's just copy-pasted from the existing code, I just moved it from the landing-page scraper to the bill page scraper.
The exception makes sense, i'll update the code and re-PR shortly.